src/utils/format.rs

```diff
@@ -0,0 +1,28 @@
+use format_num::NumberFormat;
+
+const PATTERN: &str = ".3s";
+
+pub fn si_format<T>(num: T) -> String
+where
+  T: Into<f64>,
+{
+  let fmt = NumberFormat::new();
+  fmt.format(PATTERN, num)
+}
+
+pub fn si_format_i64(num: i64) -> String {
+  si_format(num as f64)
+}
+
+#[cfg(test)]
+mod tests {
+  use crate::utils::format::si_format;
+
+  #[test]
+  fn format_check() {
```

Comment:
How does this look when formatting single, double, or triple digit numbers, e.g. 5, 26, 123? Also, you could probably test using the i64 version.

Reply:
Looks like unfortunately it doesn't remove the extra decimals in a way that the `compact` setting of the `intl` does. IE so 5 -> `5.00`